drivers/sensor/fxos8700/fxos8700.c

```diff
@@ -358,6 +366,7 @@ static const struct sensor_driver_api fxos8700_driver_api = {
 	.channel_get = fxos8700_channel_get,
 #if CONFIG_FXOS8700_TRIGGER
 	.trigger_set = fxos8700_trigger_set,
+	.attr_set = fxos8700_attr_set,
```

Comment:
Why is this inside the trigger conditional?

Reply:
fxos8700_attr_set os located in fxos8700_trigger.c and will be build only for trigger mode.
I also do not know for what it would be necessary in not trigger mode.